react/modules/graphiql.md

```diff
@@ -0,0 +1,152 @@
+#  GraphiQL
+
+**GraphiQL** is a playground/IDE for graphQL requests.
+PLEASE, READ TASK DESCRIPTION CAREFULLY UP TO THE END BEFORE STARTING THE TASK
+
+## Theoretical Notes
+
+GraphiQL is an open-source tool which you can use as an example (https://www.npmjs.com/package/@graphiql/react).
+However, your app will also include authorization/authentication capabilities, to give access to the tool to only to authorized users.
+
+## Building a team
+
+- you should work in a team of 3 members
+- teams can be pre-defined there. (https://rss-teams.web.app/)
+- All students should be registered there. (https://rss-teams.web.app/)
+- All teams should be registered there. [RSSTeams](https://rss-teams.web.app/) 
```

Comment:
Are we actually going to use this platform (rss-teams.web.app) to register teams? Because the backend doesn't work.
And I heard that such functionality will soon be added directly to the rs app

Reply:
> 

I think we should ask @valerydluski 
We tested internal teams registration a week ago

Reply:
(https://rss-teams.web.app/)  is dead